ee/docker-compose.ch.hobby.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+
+services:
+    db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
+        ports:
+            - '5432:5432'
+        restart: on-failure
+    redis:
+        image: 'redis:alpine'
+        ports:
+            - '6379:6379'
+        restart: on-failure
+    clickhouse:
+        # KEEP CLICKHOUSE-SERVER VERSION IN SYNC WITH
+        # https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/templates/clickhouse_instance.yaml#L88
+        image: yandex/clickhouse-server:21.6.5
+        depends_on:
+            - kafka
+            - zookeeper
+        ports:
+            - '8123:8123'
+            - '9000:9000'
+            - '9440:9440'
+            - '9009:9009'
+        volumes:
+            - ./idl:/idl
+            - ../docker/clickhouse/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+            - ../docker/clickhouse/config.xml:/etc/clickhouse-server/config.xml
+            - ../docker/clickhouse/users.xml:/etc/clickhouse-server/users.xml
+        restart: on-failure
+    zookeeper:
+        image: wurstmeister/zookeeper
+        restart: on-failure
+    kafka:
+        image: wurstmeister/kafka
```

Comment:
Ditto https://hub.docker.com/r/bitnami/kafka. We should also run a specific version, not just latest